wds-disable-pingbacks.php

```diff
@@ -35,11 +41,47 @@ function wds_disable_pingbacks( $methods ) {
  */
 function wds_disable_pingbacks_options( $screen ) {
 	if ( 'options-discussion' === $screen->id ) {
-		wp_register_script( 'wds-disable-pingbacks', plugins_url( 'wds-disable-pingbacks.js', __FILE__ ), 'jquery' );
-		$translations = array(
+		wp_register_script( 'wds-disable-pingbacks', plugins_url( 'wds-disable-pingbacks.js', __FILE__ ), 'jquery' ); // @codingStandardsIgnoreLine: Leave in header.
+
+		$translations = [
 			'disabled_message' => __( '(This has been globally disabled by the WDS Disable Pingbacks plugin.)', 'wds-disable-pingbacks' ),
-		);
+		];
+
 		wp_localize_script( 'wds-disable-pingbacks', 'wds_disable_pingbacks', $translations );
 		wp_enqueue_script( 'wds-disable-pingbacks' );
 	}
 }
+
+/**
+ * Ensure DOC headers are not recognized.
+ *
+ * @author Aubrey Portwood <aubrey@webdevstudios.com>
+ * @since  1.1.0
+ *
+ * @param array $headers The headers.
+ * @param mixed $wp_query Unused.
+ *
+ * @return array Modified headers.
+ */
+function wds_disable_pingbacks_modify_wp_headers( $headers, $wp_query ) {
```

Comment:
Since you don't require `$wp_query`, I don't think you need to have it accepted as an argument. Should be able to set `1` as the number of arguments.

Reply:
✅